tests/integration/without-manual-seal.test.ts

```diff
@@ -0,0 +1,59 @@
+import { expect } from 'chai'
+import { type StartedTestContainer } from 'testcontainers'
+
+import { loadProcesses } from '../../src/lib/process/index.js'
+import { multiple } from '../fixtures/programs.js'
+
+import { getVersionHelper } from '../helpers/substrateHelper.js'
+
+import { withSubstrateNode } from '../helpers/containers.js'
+
+describe('Transactions without manual seal', function () {
```

Comment:
what is a manual seal out of interest

Reply:
so the node generally produces blocks every 6s. These are then finalised usually after another 6s-18s. Only then is the transaction final. Manual seal allows us to call an API on the node and manually finalise the blocks. This makes the tests much much faster but was masking this issue. Hence I've added one test not using manual seal to catch further issues down the road